### PR TITLE
Chart values

### PR DESCRIPTION
**Pull request by xh3b4sd**
Currently based on https://github.com/giantswarm/azure-operator/pull/382. Once this is merged I will rebase on master.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -54,8 +54,8 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/e2eclients"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "azure-config-e2e"
```
Comment (xh3b4sd) [inline Gopkg.toml]: See https://github.com/giantswarm/e2etemplates/pull/37.

File: integration/setup/setup.go [hunk 2]
```diff
@@ -158,8 +157,30 @@ func Resources(config Config) error {
...
+				CalicoSubnetCIDR: env.AzureCalicoSubnetCIDR(),
+				CIDR:             env.AzureCIDR(),
+				Location:         env.AzureLocation(),
+				MasterSubnetCIDR: env.AzureMasterSubnetCIDR(),
+				VPNSubnetCIDR:    env.AzureVPNSubnetCIDR(),
+				WorkerSubnetCIDR: env.AzureWorkerSubnetCIDR(),
+			},
+			ClusterName:               env.ClusterID(),
+			CommonDomain:              env.CommonDomain(),
+			CommonDomainResourceGroup: "godsmack",
```
Comment (xh3b4sd) [inline integration/setup/setup.go]: I am not sure what to do with this. Having installation specific defaults in the templates repo feels kind of wrong.

File: integration/template/azure_config_e2e_chart_values.go [hunk 3]
```diff
@@ -1,17 +0,0 @@
-package template
```
Comment (xh3b4sd) [inline integration/template/azure_config_e2e_chart_values.go]: Note the `template` package got removed here.

File: integration/setup/setup.go (hunk 2 above)
Comment (tuommaki) [reply integration/setup/setup.go]: Could installation specific logic be in `e2e-harness` or `e2esetup`?

File: integration/env/azure.go [hunk 4]
```diff
@@ -102,27 +108,18 @@ func init() {
 			panic(err)
 		}
 
-		os.Setenv(EnvVarAzureCIDR, subnets.Parent.String())
-		os.Setenv(EnvVarAzureCalicoSubnetCIDR, subnets.Calico.String())
-		os.Setenv(EnvVarAzureMasterSubnetCIDR, subnets.Master.String())
-		os.Setenv(EnvVarAzureVPNSubnetCIDR, subnets.VPN.String())
-		os.Setenv(EnvVarAzureWorkerSubnetCIDR, subnets.Worker.String())
```
Comment (kopiczko) [inline integration/env/azure.go]: feelsgood

File: integration/setup/setup.go (hunk 2 above)
Comment (kopiczko) [reply integration/setup/setup.go]: This should be an env var IMO as the CommonDomain is.

**Pull request merged**